docs/content/preview/drivers-orms/smart-drivers.md

```diff
@@ -186,17 +186,15 @@ YugabyteDB Aeon clusters also support topology-aware load balancing. If the clus
 
 ### Deploying applications
 
-To take advantage of smart driver load balancing features when connecting to clusters in YugabyteDB Aeon, applications using smart drivers must be deployed in a VPC that has been peered with the cluster VPC. For information on VPC peering in YugabyteDB Aeon, refer to [VPC network](../../yugabyte-cloud/cloud-basics/cloud-vpcs/).
+To take advantage of smart driver load balancing features when connecting to clusters in YugabyteDB Aeon, applications using smart drivers must be deployed in a VPC that has been peered with the cluster VPC. For information on VPC peering in YugabyteDB Aeon, refer to [VPC network](../../yugabyte-cloud/cloud-basics/cloud-vpcs/). For applications that access the cluster from outside the peered network or using private endpoints via a private link, use the upstream PostgreSQL driver instead; in this case, the cluster performs the load balancing.
```

Comment:
Even in such cases, we should still encourage users to use yugabytedb drivers instead of postgresql, IMO. They can choose to disable the connection load-balancing for these cases.